File: EmployeeTODoTask/EmployeeTODoTask/Model/Employee.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace EmployeeTODoTask.Model
{
    [Table("emp_record")]
    public class Employee
    {
        [Key]
        public long id { get; set; }
        public string emp_name { get; set; }
    }

    [Table("emp_task")]
    public class EmployeeTask
    {
        [Key]
        public long id { get; set; }
        public long emp_id { get; set; }
        public string title { get; set; }
        public string description { get; set; }
        public long prioritylevel_id { get; set; }
        public long state_id { get; set; }
        public string estimate { get; set; }
    }

    [Table("priority_level")]
    public class Prioritylevel
    {
        [Key]
        public long id { get; set; }
        public string priority_level { get; set; }
    }

    [Table("statetbl")]
    public class State
    {
        [Key]
        public long id { get; set; }
        public string state_name { get; set; }
    }

}

File: EmployeTodoUI/src/app/models/Employee.ts
export class Employee {
  id: number;
  emp_name: string;
}


export class EmployeeTask {
  id: number;
  emp_id: number;
  title: string;
  description: string;
  prioritylevel: string;
  state: string;
  estimate: string;
  prioritylevel_id: number;
  state_id: number;
}

export class PriorityOrState {
  prioritylevels = new Array<Prioritylevel>();
  states = new Array<State>();
}

export class Prioritylevel {
  id: number;
  priority_level: string;
}

export class State {
  id: number;
  state_name: string;
}